### ISSUE TITLE
Add a testing table in markdown/docs to show implementation testing against `PyTorch` and `TF`

### ISSUE THREAD

**Issue by ASEM000** [ASEM000/serket#16]

**Comment by ASEM000** [comment]
prefer keras core with jax backend as in how conv, batchnorm is tested
